client/app/bundles/comments/components/CommentBox/CommentList/CommentList.spec.jsx

```diff
@@ -28,7 +28,7 @@ describe('CommentList', () => {
       id: 2,
       author: 'Furter',
       text: 'ho',
-    }),
+    })
```

Comment:
The linter failed on the trailing comma when I ran locally, with latest code, with just a bundle and npm install and rake ci.  If it doesn't for you or on travis, then there's something non-deterministic that is bleeding in from those environments or mine, and that should be locked down / prevented.  I didn't dig into why, I just made this change and the linter passed.